File: 12_CppBasics/16_FinalTask (ctrl)/Person.cpp
#include "Person.h"



using std::string;


Person::Person(const string& name, const string& surname, int age) : name(name), surname(surname), age(age) {}

void Person::setName(const string& name) {
    this->name = name;
}

void Person::setSurName(const string& surname) {
    this->surname = surname;
}

void Person::setAge(int age) {
    this->age = age;
}

string Person::getName() const {
    return name;
}

string Person::getSurName() const {
    return surname;
}

int Person::getAge() const {
    return age;
}

File: 09_JavaBasics/JavaBasics/Lesson_6/RolesBank/Person.java
package JavaBasics.Lesson_6.RolesBank;

public abstract class Person implements Operation {
    private String name;
    private String surname;


    public String getName() {
        return name;
    }

    public String getSurname() {
        return surname;
    }


    public Person(String name, String surname) {
        this.name = name;
        this.surname = surname;
    }

    public abstract void personInfo();


}
